streaming/java/dependencies.bzl

```diff
@@ -9,6 +9,7 @@ def gen_streaming_java_deps():
             "com.github.davidmoten:flatbuffers-java:1.9.0.1",
             "com.google.protobuf:protobuf-java:3.8.0",
             "de.ruedigermoeller:fst:2.57",
+            "org.aeonbits.owner:owner:1.0.10",
```

Comment:
Ray core uses `com.typesafe:config:1.3.2` for configuration. Can we use also use that in streaming? So we don't need to add this extra dependency.